7-gce/gce/startup-script.sh

```diff
@@ -56,13 +57,13 @@ chown -R pythonapp:pythonapp /opt/app
 cat >/etc/supervisor/conf.d/python-app.conf << EOF
 [program:pythonapp]
 directory=/opt/app/7-gce
-command=/opt/app/7-gce/env/bin/gunicorn main:app --bind 0.0.0.0:8080
+command=/opt/app/7-gce/env/bin/honcho start -f ./procfile worker bookshelf
```

Comment:
Why has this command changed so significantly?

Reply:
Bookshelf does 2 things: 
1 Allows you to add/review books
1 Kicks a background task to go and make an API call to find more details about the book you just added.

It looks like this demo was designed for App Engine which has tasks. But the GCE part looks like it was added after the fact, and does not support the background task. So, I added instructions to run the background task manually in the manual part, and invoked honcho to run both tasks automatically when running on GCE. I stole the honcho idea from the [App Engine flex Pub/Sub section](https://cloud.google.com/python/getting-started/using-pub-sub) and it's always the approach used by the K8s way too (though that seems added after the fact too). Using honcho like this makes the GCE part more parallel/uniform with the AE flex/K8s ways. Also just makes a feature work (book lookup) that wasn't working in the GCE way.